src/CairoMakie.jl

```diff
@@ -817,23 +814,34 @@ function AbstractPlotting.backend_show(x::CairoBackend, io::IO, ::MIME"applicati
     return screen
 end
 
+
+function AbstractPlotting.backend_show(x::CairoBackend, io::IO, ::MIME"application/postscript", scene::Scene)
+    screen = CairoScreen(scene, io; mode=:eps)
```

Comment:
isn't the pt_per_unit thing missing here?

Reply:
Yes, good point.  Will add that in.